src/components/DoctorCard/PageInfo/index.js

```diff
@@ -75,6 +74,19 @@ const PageInfo = function PageInfo({ doctor }) {
     );
   }
 
+  const websites = doctor.website
+    ?.split(',')
+    .map(w => {
+      const url = w?.trim();
+      if (url.startsWith('www')) {
+        return new URL(`http://${url}`);
+      }
+      if (url.startsWith('http')) {
+        return new URL(url);
+      }
+      return null;
+    })
+    .filter(w => Boolean(w));
   const phones = doctor.phone?.split(',');
```

Comment:
```suggestion
  const phones = doctor.phone
    ?.split(',')
    .map(p => p.trim() && new URL(`tel:${p.trim()}`))
    .filter(w => Boolean(w));

```

You can do the same with phone numbers. 
And than the DoctorsLink can be more generic
something like:

![image](https://user-images.githubusercontent.com/36345162/148671785-957b6b0e-d805-437a-9164-e67fff9fea2d.png)